### PR TITLE
22). Add topic files 59 to 62

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. 

Please merge #17732 and #17772 before merging this.

cc: @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by BillWagner** [comment]
@jameshkramer This currently has merge conflicts. Do you want me to review now, or wait until those are resolved?

**Comment by jameshkramer** [comment]
I resolved the conflicts, but we will probably get more when the prior PR, #17772, is merged. 

We get these merge conflicts because I always add redirections to the redirections file from linq-to-xml, rather than to the one in my most recent PR. We get the conflicts when there is more than one unmerged PR. We could reduce conflicts if I always built the redirections file from the one in the prior PR, even if the PR has not yet been merged. 

What do you think?

File: docs/standard/linq/control-type-projection.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+GREAL:Great Lakes Food Market:Howard Snyder
+HUNGC:Hungry Coyote Import Store:Yoshi Latimer
+LAZYK:Lazy K Kountry Store:John Steel
+LETSS:Let's Stop N Shop:Jaime Yorres
+```
+
+## See also
+
+- <xref:System.Linq.Enumerable.Select%2A>
+- [Projections and Transformations (LINQ to XML) (Visual Basic)](../../visual-basic/programming-guide/concepts/linq/projections-and-transformations-linq-to-xml.md)
```
Comment (mairaw) [inline docs/standard/linq/control-type-projection.md:135]: I imagine this article would be moved to this folder too, right?

File: docs/standard/linq/work-dictionaries-linq-xml.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 57bcefe3-8433-4d3b-935a-511c9bcbdfa8
+---
+
+# How to work with dictionaries using LINQ to XML
+
+It is often convenient to convert data structures of various kinds to XML, and then from XML to other data structures. This article shows a conversion of a <xref:System.Collections.Generic.Dictionary%602> to XML and back.
```
Comment (mairaw) [inline docs/standard/linq/work-dictionaries-linq-xml.md:13]:
```suggestion
It's often convenient to convert data structures of various kinds to XML, and then from XML to other data structures. This article shows a conversion of a <xref:System.Collections.Generic.Dictionary%602> to XML and back.
```

**Review by mairaw — COMMENTED** [review]
I imagine we'll have a few merge conflicts to deal with after the other PR is merged. But left a few comments for now.

File: docs/standard/linq/control-type-projection.md (hunk 1 above)
Comment (jameshkramer) [reply docs/standard/linq/control-type-projection.md:135]: It just becomes a parent node. You make a good point, though. That does happen in this part of the project, and I could deal with it now.

Comment (mairaw) [reply docs/standard/linq/control-type-projection.md:135]: Up to you. We can merge as-is and then once you redirect, you'd have to change these links.

**Comment by mairaw** [comment]
Some merge conflicts to resolve now 😄

**Review by mairaw — APPROVED** [review]

**Pull request merged**